information_for_authors.md

```diff
@@ -0,0 +1,3 @@
+**To get started with our template, clone this repository or click the "clone or download button" and download a zip file. Then navigate to the `templates` directory, copy the files to where you will be working, and get started!**
+
+**After forking, edit `your.README.md` and move it to `READ.md`, and you are set to write the article in your own repostitory. You can also delete the `example.tex` files that don't match your article type.
```

Comment:
Maybe add a link to Markdown format info? A naive author may not even know these are Markdown format (I've had people contribute to mine who think these are just plain text).